x-pack/plugins/apm/public/components/routing/service_detail/index.tsx

```diff
@@ -320,18 +321,29 @@ export const serviceDetail = {
           showKueryBar: false,
         },
       }),
-      '/services/{serviceName}/infrastructure': page({
-        tab: 'infrastructure',
-        title: i18n.translate('xpack.apm.views.infra.title', {
-          defaultMessage: 'Infrastructure',
+      '/services/{serviceName}/infrastructure': {
+        ...page({
+          tab: 'infrastructure',
+          title: i18n.translate('xpack.apm.views.infra.title', {
+            defaultMessage: 'Infrastructure',
+          }),
+          element: <InfraOverview />,
+          searchBarOptions: {
+            showKueryBar: false,
+            showTimeComparison: false,
```

Comment:
`showTimeComparison` and `showTransactionTypeSelector` are `false` by default.
Maybe we don't need to explicitly put them here.